packages/react-lowcode/src/codegen/app.ts

```diff
@@ -0,0 +1,72 @@
+import { AppGenerator } from './generation/generators/app-generator'
+import { UiFramework, TableType, Formatter } from './definition/context-types'
+import { CodeDir, CodeRW } from '../io'
+import ts, { factory } from "typescript"
+import { Project } from "ts-morph"
+import { CodegenOptions } from './interfaces'
+import TemplateResolver from './generation/generators/template/template-resolver'
+
+// generates CRUD React pages (master-detail, eg. orders list, order detail form) from typescript
+export function generatePages(inputSourceCode: string, io: CodeRW & CodeDir, options?: CodegenOptions) {
```

Comment:
@mat-app, this function is too hard to read. I prefer this code style with many small functions (function names document the code): https://github.com/iteria-app/lowcode/pull/128/files#diff-12351facad565720471625edf341a6daf220a8537fd080278a48dbc5e0b46380R43